tasks/base/general/configure_docker.yml

```diff
@@ -52,3 +52,8 @@
     name: docker
     enabled: yes
     daemon_reload: yes
+
+- name: Start the docker service
```

Comment:
I thinks this won't have the desired effect as we are rebooting the instance right after: https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/0624ea55ce5259b2eef061507b61956d0ef74dcf/tasks/direct-install/main.yml#L5

Having docker daemon enabled should be enough. However, if docker takes too long to start or fails to start (conf problem for example) it might be interesting to ensure that docker is up right before running the actual ECE install. 

Moving this current piece of code right before might be what we need
https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/0624ea55ce5259b2eef061507b61956d0ef74dcf/tasks/ece-bootstrap/main.yml#L21

Also, if anyone want to use this playbook to only bootstrap ECE (tag `bootstrap`), i.e. on a machine which was already provisioned (docker installed, configured) , that also might be interesting to ensure docker is up before starting the actual install of ECE.

Reply:
I like your suggestion for the alternative location for the docker start task, I'll go ahead and test that change in my environment, just to make sure it functions as intended!